airflow/models/baseoperator.py

```diff
@@ -220,14 +220,14 @@ class derived from this one results in the creation of a task object,
     # Defines which files extensions to look for in the templated fields
     template_ext = []  # type: Iterable[str]
     # Defines the color in the UI
-    ui_color = '#fff'
-    ui_fgcolor = '#000'
+    ui_color = '#fff'  # type str
+    ui_fgcolor = '#000'  # type str
 
     # base list which includes all the attrs that don't need deep copy.
     _base_operator_shallow_copy_attrs = ('user_defined_macros',
                                          'user_defined_filters',
                                          'params',
-                                         '_log',)
+                                         '_log',)  # type: Iterable[str]
```

Comment:
Shouldn't it be a `Tuple[str]`?

Reply:
It could be `Tuple` however usage is not restricted to not mutable types, hence I used `Iterable`